koinos/contracts/standards/kcs1.proto

```diff
@@ -0,0 +1,80 @@
+syntax = "proto3";
+
+package koinos.contracts.standards.kcs1;
```

Comment:
I agree it is important to have each contract in its own namespace to avoid name collisions, this does result in some long namespaces. In C++, for example, the namespace is `koinos::contracts::standards::kcs1`.

In light of this requirements, I am considering dropping `contracts` from the path.

This would mean `koinos/contracts` is reserved for proto files relating to specific system contracts and `koinos/standards` is related only contract standards.

Does this make sense? Is it a reasonable and meaningful distinction?

Another possibility could be dropping `standards` from the namespace and simply having `koinos/contracts/kcs1`. Does it make sense to have standard contracts and system contracts within the same namespace or is this dirty?

Reply:
How about `koinos/standard/kcs1.proto`?